.stylelintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "rules": {
+    "block-no-empty": null,
+    "no-empty-source": null,
+    "no-descending-specificity": null
```

Comment:
I don't have opinions on these, but the linter didn't like them, so I just disabled to keep the current standards in place.